service/models/operations/calibrate.py

```diff
@@ -25,19 +25,12 @@ class CalibrateExtra(BaseModel):
     num_iterations: int = Field(
         1000, description="Optional field for CIEMSS calibration", example=1000
     )
-    lr: float = Field(
-        0.03, description="Optional field for CIEMSS calibration", example=0.03
-    )
     verbose: bool = Field(
         False, description="Optional field for CIEMSS calibration", example=False
     )
     num_particles: int = Field(
         1, description="Optional field for CIEMSS calibration", example=1
     )
-    # autoguide: pyro.infer.autoguide.AutoLowRankMultivariateNormal
-    solver_method: str = Field(
-        "dopri5", description="Optional field for CIEMSS calibration", example="dopri5"
-    )
```

Comment:
Why are we making these changes? Have we consider the impact and the types of corresponding changes we need to make?

Reply:
These changes are a result of the changes requested here: https://github.com/DARPA-ASKEM/terarium/issues/4201